### PR TITLE
Add chat.test.js

### LINKED ISSUE #75: Chat Test

### PR DESCRIPTION
**Pull request by Chriscastanuela**
## Description
* Tests. `Chat.js`

## Where To Start
* `Chat.js`

## Affected areas of application
`Chat.test.js`

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Test A - jest
- [] Test B

## Relevant Tickets
Closes #75 

## Tags
@holladayian 
@GreyMatteOr

### CODE REVIEW

File: src/Chat/Chat.js [hunk 1]
```diff
@@ -10,16 +10,16 @@ export class Chat extends Component {
       newMessage: ''
     }
     this.messagesEnd = React.createRef()
+    if (typeof(module) != 'undefined') {
+      this.messagesEnd.current = {}
+      this.messagesEnd.current.scrollIntoView = () => {}
+    }
```
Comment (holladayian) [inline src/Chat/Chat.js]: I might have to go over this again with one of y'all

File: src/Chat/Chat.test.js [hunk 2]
```diff
@@ -1,118 +1,47 @@
...
+
+  describe('Functionality', () => { 
+    it('3. should send chats', () => {
+
+            let mockFn = jest.fn();
+            render(<Chat
+                chat={['chris, matt, ian']}
+                sendMessage={mockFn}
+            />);
+            userEvent.type(screen.getByPlaceholderText('Chat'), 'Chris')
```
Comment (GreyMatteOr) [inline src/Chat/Chat.test.js:38]: WHOA! Nice

File: src/Chat/Chat.test.js [hunk 3]
```diff
@@ -1,118 +1,47 @@
...
-//       expect(screen.getByText('two')).toBeInTheDocument();
-//       expect(screen.getByText('three')).toBeInTheDocument();
-//     })
-//   })
-// })
+describe('Inputs exist', () => {
+
+  describe(' `Chat` input ', () => {
+    it('1. should render the `Chat` input`', () => {
+        let mockFn = jest.fn();
```
Comment (GreyMatteOr) [inline src/Chat/Chat.test.js:11]: Looks like prime beforeEach fodder

**Review by GreyMatteOr — APPROVED** [review]

File: src/Chat/Chat.test.js (hunk 3 above)
Comment (holladayian) [inline src/Chat/Chat.test.js:11]: We might be able to give this one a bit more proper of a name. Man, I feel so British saying this.

**Review by holladayian — APPROVED** [review]

**Pull request merged**